rs/sns/governance/api/src/ic_sns_governance.pb.v1.rs

```diff
@@ -1645,6 +1645,33 @@ pub struct GetMetadataResponse {
     pub name: Option<String>,
     pub description: Option<String>,
 }
+/// Request message for 'get_metadata'.
+#[derive(Default, candid::CandidType, candid::Deserialize, Debug, Clone, Copy, PartialEq)]
+pub struct GetMetricsRequest {
+    pub time_window_seconds: Option<u64>,
+}
+
+pub mod get_sns_status_response {
+    use super::GovernanceError;
+
+    #[derive(Default, candid::CandidType, candid::Deserialize, Debug, Clone, PartialEq)]
+    pub struct Metrics {
+        pub num_recent_proposals: Option<u64>,
+        pub last_transaction_timestamp: Option<u64>,
+    }
+
+    #[derive(candid::CandidType, candid::Deserialize, Debug, Clone, PartialEq)]
+    pub enum GetMetricsResult {
```

Comment:
I'm curious if we could just use `Result<Metrics, GovernanceError>` in place of this. Or would the Candid check fail?

Reply:
NCR as we already have a similar pattern in this canister.